File: src/main/java/com/kamegatze/learnjvm/controllers/account/AccountInfoController.java
package com.kamegatze.learnjvm.controllers.account;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/account")
public class AccountInfoController {

    @GetMapping("/page_info")
    public String handlingAccount() {return "account/main";}

}

File: src/main/kotlin/com/kamegatze/learnjvm/controllers/account/AccountInfoController.kt
package com.kamegatze.learnjvm.controllers.account

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping


@Controller
@RequestMapping("/account")
class AccountInfoController {

    @GetMapping("/page_info")
    fun handlingAccount() = "account/main"

}